pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/impl/BloomFilterRule.java

```diff
@@ -41,6 +45,8 @@
 public class BloomFilterRule extends AbstractRule {
   private static final Logger LOGGER = LoggerFactory.getLogger(BloomFilterRule.class);
   private final BloomFilterRuleParams _params;
+  private static final Set<DataType> COMPATIBLE_DATA_TYPES = new HashSet<>(
+      Arrays.asList(DataType.INT, DataType.LONG, DataType.FLOAT, DataType.DOUBLE, DataType.STRING, DataType.BYTES));
```

Comment:
Assumed these are the compatible types after looking at BloomFilterHandler.java

Reply:
(nit) you want to add this as comment as well?